docs/version-check

```diff
@@ -27,11 +27,9 @@ if [[ -n "$CIRRUS_PR" ]]; then
     exit 0
 fi
 
-if [[ -n "$CIRRUS_CRON" ]]; then
-    if [[ "$CIRRUS_CRON" != "nightly" ]]; then
-        warn "Only meaningful when CIRRUS_CRON=nightly (it is '$CIRRUS_CRON')"
-        exit 0
-    fi
+if [[ -z "$CIRRUS_CRON" ]] || [[ "$CIRRUS_CRON" != "main" ]]; then
+    warn "Only meaningful when CIRRUS_CRON=main (it is '$CIRRUS_CRON')"
+    exit 0
```

Comment:
Uh, no, this breaks the "run me interactively" part.

Too much churn, so I won't offer a suggestion. Once this stabilizes, I can take a new look.

Reply:
Sorry I don't follow.  My assumption is the script should only run in full when triggered by cirrus-cron for the `main` job (i.e. `$CIRRUS_CRON=="main"` - the branch and job name happen to be the same) and never any other time.  Is there some other condition where it should also operate in some way?

Reply:
OIC, you mean when run by a human.  Gotcha, I can fix that.